.goxc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "ArtifactsDest": "dist",
+    "Tasks": [
+        "clean-destination",
+        "xc",
+        "archive-zip",
+        "rmbin"
+    ],
+    "TaskSettings": {
+        "archive-zip": {
+            "include-top-level-dir": "windows darwin linux",
+            "platforms": "windows darwin linux"
+        }
+    },
+    "Arch": "386 amd64",
+    "Os": "linux darwin windows",
```

Comment:
I don't know whether this plugin works in Windows. I hope, but maybe never tested.

Reply:
I'll check it.